src/_posts/compliance/security-overview/2000-01-01-employee-security.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Employee Security
+nav: Employees
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures
+index: 9
+---
+
+### Security Training
+
+All employees receive security training to ensure that they are aware of the security measures implemented by the
+platform and to prevent security incidents caused by human error.
+
+### Security Awareness
+
+All employees are required to follow security best practices to protect the platform from security incidents. This
+includes using strong passwords, enabling two-factor authentication, and following the security policies implemented by
+the platform.
+
+They are trained to detect phishing attempts and to report them to the security team.
```

Comment:
Be more inclusive, like "to detect phishing, social engineering, or impersonation" ?

Reply:
ack